packages/world/src/systems/MineSystem.sol

```diff
@@ -174,6 +174,13 @@ contract MineSystem is System {
       return NatureLib.growSeed(coord, mined, minedType);
     }
 
+    if (Mass._get(mined) == 0) {
+      // If the mass is 0, we assume the block was not correctly setup (e.g. missing mass)
+      // NOTE: This currently targets the issue where grown seeds/saplings were not given mass
+      // TODO: We could potentially stop assigning mass on build and just do it here
```

Comment:
Would just move the gas from build to mine right? or are u thinking there'd be some other benefit

Reply:
yeah no real additional benefit, just less gas when calling getOrCreateBlock, etc